src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -398,6 +418,8 @@ void activity_classifier::train(
   state_update["training_time"] = time_object.current_time();
 
   add_or_update_state(state_update);
+  std::cout << "Training complete" << std::endl;
```

Comment:
We should be using `logprogress_stream` instead of `std::cout` for this kind of thing, I believe. See examples in OD and in supervised learning, e.g. logistic regression